src/components/calcite-radio/calcite-radio.tsx

```diff
@@ -19,13 +19,19 @@ export class CalciteRadio {
   /** The disabled state of the radio. */
   @Prop({ reflect: true }) disabled?: boolean = false;
 
-  /** The focused state of the radio. */
+  /**
+   * The focused state of the radio.
+   * @private
+   */
   @Prop({ reflect: true }) focused = false;
```

Comment:
I think this is gone in latest branch.

Reply:
This one is still in master. cc @eriklharper